log/logger_impl.go

```diff
@@ -13,13 +13,26 @@ import (
 )
 
 func NewDefaultLogger() Logger {
-	return NewLogger(log.NewLogfmtLogger(log.NewSyncWriter(os.Stderr)))
+	switch strings.ToLower(os.Getenv("MOOV_LOG_FORMAT")) {
```

Comment:
Do we need a Moov prefix?

Reply:
It is a MOOV library and wouldn't want to interfere with anyone else using say a LOG_FORMAT value that doesn't match what we expect.

Reply:
So, the default logger is not a default logger and it can be one of?

On one hand, I think it's an easy and quick solution with no code changes required. On the other hand, it changes the behavior of the "Default" logger. It can be a breaking change for tests.

We don't have a lot of places where we instantiate a logger in our projects (`environment.go`, and maybe some other). Maybe we can create a new method and update `templater` instead?